diff -up cppcheck-1.52/test/testfilelister.cpp.orig cppcheck-1.52/test/testfilelister.cpp
--- cppcheck-1.52/test/testfilelister.cpp.orig	2011-12-10 14:00:45.000000000 +0200
+++ cppcheck-1.52/test/testfilelister.cpp	2012-01-05 12:45:09.197377791 +0200
@@ -24,6 +24,7 @@
 #ifndef _WIN32
 #include <limits.h>
 #include <stdlib.h>
+#include <unistd.h>
 #endif
 
 class TestFileLister: public TestFixture {
