diff -up cppcheck-1.90/htmlreport/cppcheck-htmlreport.orig cppcheck-1.90/htmlreport/cppcheck-htmlreport
--- cppcheck-1.90/htmlreport/cppcheck-htmlreport.orig	2019-12-21 11:51:10.000000000 +0100
+++ cppcheck-1.90/htmlreport/cppcheck-htmlreport	2019-12-22 10:56:52.071627890 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 from __future__ import unicode_literals
 
diff -up cppcheck-1.90/htmlreport/setup.py.orig cppcheck-1.90/htmlreport/setup.py
--- cppcheck-1.90/htmlreport/setup.py.orig	2019-12-21 11:51:10.000000000 +0100
+++ cppcheck-1.90/htmlreport/setup.py	2019-12-22 10:56:51.984631669 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 from setuptools import setup
 
diff -up cppcheck-1.90/htmlreport/test_htmlreport.py.orig cppcheck-1.90/htmlreport/test_htmlreport.py
--- cppcheck-1.90/htmlreport/test_htmlreport.py.orig	2019-12-21 11:51:10.000000000 +0100
+++ cppcheck-1.90/htmlreport/test_htmlreport.py	2019-12-22 10:56:52.012630453 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 """Test cppcheck-htmlreport."""
 
 import os
