diff -up cppcheck-1.75/cli/CMakeLists.txt.tinyxml cppcheck-1.75/cli/CMakeLists.txt
--- cppcheck-1.75/cli/CMakeLists.txt.tinyxml	2016-08-09 06:43:08.152146498 +0200
+++ cppcheck-1.75/cli/CMakeLists.txt	2016-08-09 06:44:12.819146690 +0200
@@ -1,5 +1,4 @@
 include_directories(${PROJECT_SOURCE_DIR}/lib/)
-include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/externals/tinyxml/)
 include_directories(${PROJECT_SOURCE_DIR}/externals/simplecpp/)
 
 file(GLOB hdrs "*.h")
@@ -8,10 +7,11 @@ file(GLOB mainfile "main.cpp")
 list(REMOVE_ITEM srcs ${mainfile})
 
 add_library(cli_objs OBJECT ${hdrs} ${srcs}) 
-add_executable(cppcheck ${hdrs} ${mainfile} $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:tinyxml_objs> $<TARGET_OBJECTS:simplecpp_objs>)
+add_executable(cppcheck ${hdrs} ${mainfile} $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:simplecpp_objs>)
 if (HAVE_RULES)
     target_link_libraries(cppcheck pcre)
 endif()
+target_link_libraries(cppcheck tinyxml2)
 
 install(TARGETS cppcheck
     RUNTIME DESTINATION ${CMAKE_INSTALL_FULL_BINDIR}
diff -up cppcheck-1.75/CMakeLists.txt.tinyxml cppcheck-1.75/CMakeLists.txt
--- cppcheck-1.75/CMakeLists.txt.tinyxml	2016-08-09 06:43:08.153146498 +0200
+++ cppcheck-1.75/CMakeLists.txt	2016-08-09 06:43:31.191146016 +0200
@@ -16,7 +16,6 @@ if (BUILD_TESTS)
     enable_testing()
 endif()
 
-add_subdirectory(externals/tinyxml)
 add_subdirectory(externals/simplecpp)
 add_subdirectory(lib)   # CppCheck Library 
 add_subdirectory(cli)   # Client application
diff -up cppcheck-1.75/gui/CMakeLists.txt.tinyxml cppcheck-1.75/gui/CMakeLists.txt
--- cppcheck-1.75/gui/CMakeLists.txt.tinyxml	2016-08-06 18:49:40.000000000 +0200
+++ cppcheck-1.75/gui/CMakeLists.txt	2016-08-09 06:44:44.234147074 +0200
@@ -11,7 +11,6 @@ if (BUILD_GUI)
     endif()
     
     include_directories(${PROJECT_SOURCE_DIR}/lib/)
-    include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/externals/tinyxml/)
 
     file(GLOB hdrs "*.h")
     file(GLOB srcs "*.cpp")
@@ -21,12 +20,12 @@ if (BUILD_GUI)
     QT4_ADD_RESOURCES(resources "gui.qrc")
     QT4_ADD_TRANSLATION(qms ${tss})
     
-    add_executable(cppcheck-gui ${hdrs} ${srcs} ${uis_hdrs} ${resources} ${qms} $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:tinyxml_objs> $<TARGET_OBJECTS:simplecpp_objs>)
+    add_executable(cppcheck-gui ${hdrs} ${srcs} ${uis_hdrs} ${resources} ${qms} $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:simplecpp_objs>)
     target_link_libraries(cppcheck-gui ${QT_LIBRARIES})
     if (HAVE_RULES)
         target_link_libraries(cppcheck-gui pcre)
     endif()
-
+    target_link_libraries(cppcheck-gui tinyxml2)
 
     install(TARGETS cppcheck-gui RUNTIME DESTINATION ${CMAKE_INSTALL_FULL_BINDIR} COMPONENT applications)
     install(FILES ${qms} DESTINATION ${CMAKE_INSTALL_FULL_BINDIR} COMPONENT applications)
diff -up cppcheck-1.75/test/CMakeLists.txt.tinyxml cppcheck-1.75/test/CMakeLists.txt
--- cppcheck-1.75/test/CMakeLists.txt.tinyxml	2016-08-09 06:43:08.155146498 +0200
+++ cppcheck-1.75/test/CMakeLists.txt	2016-08-09 06:45:26.020153572 +0200
@@ -1,16 +1,16 @@
 if (BUILD_TESTS)
 
     include_directories(${PROJECT_SOURCE_DIR}/lib/ ${PROJECT_SOURCE_DIR}/cli/)
-    include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/externals/tinyxml)
     include_directories(${PROJECT_SOURCE_DIR}/externals/simplecpp/)
 
     file(GLOB hdrs "*.h")
     file(GLOB srcs "*.cpp")
 
-    add_executable(testrunner ${hdrs} ${srcs} $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:tinyxml_objs> $<TARGET_OBJECTS:simplecpp_objs>)
+    add_executable(testrunner ${hdrs} ${srcs} $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:simplecpp_objs>)
     if (HAVE_RULES)
         target_link_libraries(testrunner pcre)
     endif()
+    target_link_libraries(testrunner tinyxml2)
 
     add_custom_target(copy_cfg ALL
         COMMENT "Copying cfg files")
