diff -urp cppcheck-1.89.orig/man/manual.docbook cppcheck-1.89/man/manual.docbook
--- cppcheck-1.89.orig/man/manual.docbook	2019-09-01 09:01:12.000000000 -0400
+++ cppcheck-1.89/man/manual.docbook	2019-12-07 14:04:54.363409384 -0500
@@ -1217,7 +1217,7 @@ Checking pen1.c...
   &lt;/resource&gt;
 &lt;/def&gt;</programlisting>
 
-        <para>Functions that reallocate memory can be configured using a realloc tag. The input argument which points to the memory that shall be reallocated can also be configured (the default is the first argument). As an example, here is a configuration file for the fopen, freopen and fclose functions from the c standard library:
+        <para>Functions that reallocate memory can be configured using a realloc tag. The input argument which points to the memory that shall be reallocated can also be configured (the default is the first argument). As an example, here is a configuration file for the fopen, freopen and fclose functions from the c standard library:</para>
 
         <programlisting>&lt;?xml version="1.0"?&gt;
 &lt;def&gt;
