--- cppcheck-1.70/cli/CMakeLists.txt.stlink	2015-11-13 15:07:57.987502410 +0900
+++ cppcheck-1.70/cli/CMakeLists.txt	2015-11-13 16:13:30.256811541 +0900
@@ -5,7 +5,7 @@
 file(GLOB mainfile "main.cpp")
 list(REMOVE_ITEM srcs ${mainfile})
 
-link_libraries(${PROJECT_NAME} -ltinyxml2)
+link_libraries(-Wl,--whole-archive ${PROJECT_NAME} -Wl,--no-whole-archive -ltinyxml2)
 
 add_library(cli_lib OBJECT ${hdrs} ${srcs})
 
--- cppcheck-1.70/gui/CMakeLists.txt.stlink	2015-11-13 15:07:57.989502413 +0900
+++ cppcheck-1.70/gui/CMakeLists.txt	2015-11-13 16:13:47.930824027 +0900
@@ -21,7 +21,7 @@
     QT4_ADD_RESOURCES(resources "gui.qrc")
     QT4_ADD_TRANSLATION(qms ${tss})
     
-    link_libraries(${PROJECT_NAME} -ltinyxml2 ${QT_LIBRARIES})
+    link_libraries(-Wl,--whole-archive ${PROJECT_NAME} -Wl,--no-whole-archive -ltinyxml2 ${QT_LIBRARIES})
 
     add_executable(cppcheck-gui ${hdrs} ${srcs} ${uis_hdrs} ${resources} ${qms})
 
