diff -urp cppcheck-1.88.orig/cli/CMakeLists.txt cppcheck-1.88/cli/CMakeLists.txt
--- cppcheck-1.88.orig/cli/CMakeLists.txt	2019-06-29 03:51:25.000000000 -0400
+++ cppcheck-1.88/cli/CMakeLists.txt	2019-07-01 20:38:10.056775258 -0400
@@ -1,5 +1,4 @@
 include_directories(${PROJECT_SOURCE_DIR}/lib/)
-include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/externals/tinyxml/)
 include_directories(${PROJECT_SOURCE_DIR}/externals/simplecpp/)
 
 file(GLOB hdrs "*.h")
@@ -8,7 +7,7 @@ file(GLOB mainfile "main.cpp")
 list(REMOVE_ITEM srcs ${mainfile})
 
 add_library(cli_objs OBJECT ${hdrs} ${srcs}) 
-add_executable(cppcheck ${hdrs} ${mainfile} $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:tinyxml_objs> $<TARGET_OBJECTS:simplecpp_objs>)
+add_executable(cppcheck ${hdrs} ${mainfile} $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:simplecpp_objs>)
 if (HAVE_RULES)
     find_library(PCRE_LIBRARY pcre)
     target_link_libraries(cppcheck ${PCRE_LIBRARY})
@@ -18,6 +17,7 @@ endif()
 if (WIN32 AND NOT BORLAND)
     target_link_libraries(cppcheck Shlwapi.lib)
 endif()
+target_link_libraries(cppcheck tinyxml2)
 
 install(TARGETS cppcheck
     RUNTIME DESTINATION ${CMAKE_INSTALL_FULL_BINDIR}
diff -urp cppcheck-1.88.orig/CMakeLists.txt cppcheck-1.88/CMakeLists.txt
--- cppcheck-1.88.orig/CMakeLists.txt	2019-06-29 03:51:25.000000000 -0400
+++ cppcheck-1.88/CMakeLists.txt	2019-07-01 20:38:10.078775256 -0400
@@ -19,7 +19,6 @@ if (BUILD_TESTS)
     enable_testing()
 endif()
 
-add_subdirectory(externals/tinyxml)
 add_subdirectory(externals/simplecpp)
 add_subdirectory(lib)   # CppCheck Library 
 add_subdirectory(cli)   # Client application
diff -urp cppcheck-1.88.orig/gui/CMakeLists.txt cppcheck-1.88/gui/CMakeLists.txt
--- cppcheck-1.88.orig/gui/CMakeLists.txt	2019-06-29 03:51:25.000000000 -0400
+++ cppcheck-1.88/gui/CMakeLists.txt	2019-07-01 20:41:48.862751778 -0400
@@ -10,7 +10,6 @@ if (BUILD_GUI)
     endif()
 
     include_directories(${PROJECT_SOURCE_DIR}/lib/)
-    include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/externals/tinyxml/)
 
     file(GLOB hdrs "*.h")
     file(GLOB srcs "*.cpp")
@@ -20,11 +19,11 @@ if (BUILD_GUI)
     QT5_ADD_RESOURCES(resources "gui.qrc")
     QT5_ADD_TRANSLATION(qms ${tss})
 
-    add_executable(cppcheck-gui ${hdrs} ${srcs} ${uis_hdrs} ${resources} ${qms} $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:tinyxml_objs> $<TARGET_OBJECTS:simplecpp_objs>)
+    add_executable(cppcheck-gui ${hdrs} ${srcs} ${uis_hdrs} ${resources} ${qms} $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:simplecpp_objs>)
     if (HAVE_RULES)
         target_link_libraries(cppcheck-gui pcre)
     endif()
-    target_link_libraries(cppcheck-gui Qt5::Core Qt5::Gui Qt5::Widgets Qt5::PrintSupport)
+    target_link_libraries(cppcheck-gui tinyxml2 Qt5::Core Qt5::Gui Qt5::Widgets Qt5::PrintSupport)
     if(WITH_QCHART)
         target_compile_definitions (cppcheck-gui PRIVATE HAVE_QCHART )
         target_link_libraries(cppcheck-gui Qt5::Charts)
diff -urp cppcheck-1.88.orig/test/CMakeLists.txt cppcheck-1.88/test/CMakeLists.txt
--- cppcheck-1.88.orig/test/CMakeLists.txt	2019-06-29 03:51:25.000000000 -0400
+++ cppcheck-1.88/test/CMakeLists.txt	2019-07-01 20:43:12.299742824 -0400
@@ -1,16 +1,16 @@
 if (BUILD_TESTS)
 
     include_directories(${PROJECT_SOURCE_DIR}/lib/ ${PROJECT_SOURCE_DIR}/cli/)
-    include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/externals/tinyxml)
     include_directories(${PROJECT_SOURCE_DIR}/externals/simplecpp/)
 
     file(GLOB hdrs "*.h")
     file(GLOB srcs "*.cpp")
 
-    add_executable(testrunner ${hdrs} ${srcs} $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:tinyxml_objs> $<TARGET_OBJECTS:simplecpp_objs>)
+    add_executable(testrunner ${hdrs} ${srcs} $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:simplecpp_objs>)
     if (HAVE_RULES)
         target_link_libraries(testrunner pcre)
     endif()
+    target_link_libraries(testrunner tinyxml2)
 
     add_custom_target(copy_cfg ALL
         COMMENT "Copying cfg files")
