diff -up cppcheck-1.73/cli/CMakeLists.txt.link cppcheck-1.73/cli/CMakeLists.txt
--- cppcheck-1.73/cli/CMakeLists.txt.link	2016-04-10 03:25:25.329210822 +0200
+++ cppcheck-1.73/cli/CMakeLists.txt	2016-04-10 03:26:51.813213485 +0200
@@ -11,6 +11,7 @@ if (HAVE_RULES)
     target_link_libraries(cppcheck pcre)
 endif()
 target_link_libraries(cppcheck -ltinyxml2)
+set_target_properties(cppcheck PROPERTIES LINKER_LANGUAGE C++)
 
 install(TARGETS cppcheck
     RUNTIME DESTINATION ${CMAKE_INSTALL_FULL_BINDIR}
diff -up cppcheck-1.73/gui/CMakeLists.txt.link cppcheck-1.73/gui/CMakeLists.txt
--- cppcheck-1.73/gui/CMakeLists.txt.link	2016-04-10 03:25:25.322210822 +0200
+++ cppcheck-1.73/gui/CMakeLists.txt	2016-04-10 03:25:25.331210822 +0200
@@ -26,7 +26,7 @@ if (BUILD_GUI)
     if (HAVE_RULES)
         target_link_libraries(cppcheck-gui pcre)
     endif()
-
+    set_target_properties(cppcheck-gui  PROPERTIES LINKER_LANGUAGE C++)
 
     install(TARGETS cppcheck-gui RUNTIME DESTINATION ${CMAKE_INSTALL_FULL_BINDIR} COMPONENT applications)
     install(FILES ${qms} DESTINATION share/${PROJECT_NAME}/lang/ COMPONENT applications)
diff -up cppcheck-1.73/test/CMakeLists.txt.link cppcheck-1.73/test/CMakeLists.txt
--- cppcheck-1.73/test/CMakeLists.txt.link	2016-04-10 03:25:25.317210822 +0200
+++ cppcheck-1.73/test/CMakeLists.txt	2016-04-10 03:25:25.333210822 +0200
@@ -10,6 +10,7 @@ if (BUILD_TESTS)
     if (HAVE_RULES)
         target_link_libraries(testrunner pcre)
     endif()
+    set_target_properties(testrunner  PROPERTIES LINKER_LANGUAGE C++)
 
     add_custom_target(copy_cfg ALL
         COMMENT "Copying cfg files")
