diff -urp cppcheck-1.85.orig/htmlreport/cppcheck-htmlreport cppcheck-1.85/htmlreport/cppcheck-htmlreport
--- cppcheck-1.85.orig/htmlreport/cppcheck-htmlreport	2018-10-14 14:56:51.000000000 +0200
+++ cppcheck-1.85/htmlreport/cppcheck-htmlreport	2018-11-08 16:26:00.527947956 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 from __future__ import unicode_literals
 
diff -urp cppcheck-1.85.orig/htmlreport/setup.py cppcheck-1.85/htmlreport/setup.py
--- cppcheck-1.85.orig/htmlreport/setup.py	2018-10-14 14:56:51.000000000 +0200
+++ cppcheck-1.85/htmlreport/setup.py	2018-11-08 16:26:40.266947340 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 from setuptools import setup
 
diff -urp cppcheck-1.85.orig/htmlreport/test_htmlreport.py cppcheck-1.85/htmlreport/test_htmlreport.py
--- cppcheck-1.85.orig/htmlreport/test_htmlreport.py	2018-10-14 14:56:51.000000000 +0200
+++ cppcheck-1.85/htmlreport/test_htmlreport.py	2018-11-08 16:26:25.963947562 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 """Test cppcheck-htmlreport."""
 
 import os
